cdk/bin/cdk.ts

```diff
@@ -97,11 +97,13 @@ new AcquisitionEventsApi(app, "Acquisition-Events-API-PROD", {
 new BigqueryAcquisitionsPublisher(app, "BigqueryAcquisitionsPublisher-CODE", {
   stack: "support",
   stage: "CODE",
+  softOptInConsentSetterQueueArn: "arn:aws:sqs:eu-west-1:865473395570:soft-opt-in-consent-setter-queue-CODE",
```

Comment:
I think you can use something like ${aws::accountid} to avoid hard coding it here